R/plot-population-time-profile.R

```diff
@@ -54,6 +45,7 @@ plotPopulationTimeProfile <- function(dataCombined,
     dataCombined,
     defaultPlotConfiguration,
     aggregation,
-    probs = quantiles
+    probs = quantiles,
```

Comment:
I would explicitly list the new argument here, as the user would otherwise not know about what are the possible parameters.

Reply:
I added `n` to the documentation.